dashboard-app/src/dashboard/hooks/useAggregatedDataByActivity.ts

```diff
@@ -0,0 +1,74 @@
+import { DependencyList, useEffect, useState } from 'react';
```

Comment:
I think these hook names should be present tense

`useAggregateDataByActivity`